games/o.yaml

```diff
@@ -1568,6 +1568,26 @@
   video:
     youtube: Xfv_UrDq-As
 
+- name: OpenNitemare3D
+  originals:
+  - Nitemare 3D
+  type: remake
+  repo: https://github.com/BBQGiraffe/OpenNitemare3D
+  development: sporadic
+  status: semi-playable
+  content: commercial
+  lang:
+  - C#
+  - C
+  framework:
+  - SDL2
+  license:
+  - As-is
```

Comment:
Checking to see if the project should be GPL3 instead: https://github.com/BBQGiraffe/OpenNitemare3D/pull/19